src/ai/susi/server/api/aaa/LoginService.java

```diff
@@ -237,6 +241,15 @@ else if(post.get("sessionID", null) != null && post.get("response", null) != nul
 			// store the IP of last login in accounting object
 			Accounting accounting = DAO.getAccounting(identity);
 			accounting.getJSON().put("lastLoginIP", post.getClientHost());
+
+			// store the time of last login in accounting object
+			Date currentTime = new Date();
+			SimpleDateFormat lastLoginTime = new SimpleDateFormat("EEE, MMM d, yyyy hh:mm:ss a z");
```

Comment:
Please do not define the date format here. Use ai.susi.tools.DateParser.FORMAT_RFC1123 instead